libsolidity/formal/CHC.cpp

```diff
@@ -323,6 +339,71 @@ void CHC::unknownFunctionCall(FunctionCall const&)
 	m_unknownFunctionCallSeen = true;
 }
 
+void CHC::visitLoop(
+	BreakableStatement const& _loop,
+	Expression const* _condition,
+	Statement const& _body,
+	ASTNode const* _postLoop
+)
+{
+	solAssert(m_currentFunction, "");
+	auto const& functionBody = m_currentFunction->body();
+
+	createBlock(&_loop, "loop_header_");
```

Comment:
Yeah, but conceptually the nicer ASTNode to associate with the loop header would be ``_condition``. But I guess you can't do that, since ``_condition`` can be ``nullptr``?
But yeah, not too important anyways.

Reply:
Ah right, yes, I think it can be null.